test/spec/algoliasearch.helper/derive/multiqueries.js

```diff
@@ -2,48 +2,54 @@
 
 var algoliasearchHelper = require('../../../../index.js');
 
-test('[Derived helper] no derived helpers', function() {
-  expect.assertions(1);
-  var client = {
-    search: searchTest
+function makeFakeClient(assertions) {
+  return {
+    search: function() {
+      assertions.apply(null, arguments);
+
+      return new Promise(function() {});
+    }
   };
+}
+
+test('[Derived helper] no derived helpers', function() {
+  var client = makeFakeClient(assertions);
   var helper = algoliasearchHelper(client, '');
+
   helper.search();
 
-  function searchTest(requests) {
+  function assertions(requests) {
```

Comment:
I think I'd find these easier to read if they were inlined in the creation of the fake client instead of being hoisted, wdyt?

Reply:
I've kept the original implementation. At first, I've changed it but I revert it. I've found it easier like this. Most of the time we expect to have the assertions at the end of the test. I don't have a strong opinion about that though. We can change it.